x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/40_unsupported_types.yml

```diff
@@ -308,106 +308,103 @@ unsupported with sort:
+  - match: { columns.22.name: search_as_you_type._index_prefix }
   - match: { columns.22.type: unsupported }
-  - match: { columns.23.name: search_as_you_type._index_prefix }
-  - match: { columns.23.type: unsupported }
-  - match: { columns.24.name: shape }
-  - match: { columns.24.type: cartesian_shape }
-  - match: { columns.25.name: some_doc.bar }
-  - match: { columns.25.type: long }
-  - match: { columns.26.name: some_doc.foo }
-  - match: { columns.26.type: keyword }
-  - match: { columns.27.name: text }
-  - match: { columns.27.type: text }
-  - match: { columns.28.name: token_count }
-  - match: { columns.28.type: integer }
+  - match: { columns.23.name: shape }
+  - match: { columns.23.type: cartesian_shape }
+  - match: { columns.24.name: some_doc.bar }
+  - match: { columns.24.type: long }
+  - match: { columns.25.name: some_doc.foo }
+  - match: { columns.25.type: keyword }
+  - match: { columns.26.name: text }
+  - match: { columns.26.type: text }
+  - match: { columns.27.name: token_count }
+  - match: { columns.27.type: integer }
 
   - length: { values: 1 }
   - match: { values.0.0: null }
   - match: { values.0.1: null }
-  - match: { values.0.2: null }
-  - match: { values.0.3: "2015-01-01T12:10:30.123456789Z" }
+  - match: { values.0.2: "2015-01-01T12:10:30.123456789Z" }
+  - match: { values.0.3: null }
   - match: { values.0.4: null }
   - match: { values.0.5: null }
   - match: { values.0.6: null }
-  - match: { values.0.7: null }
+  - match: { values.0.7: "POINT (10.0 12.0)" }
   - match: { values.0.8: "POINT (10.0 12.0)" }
-  - match: { values.0.9: "POINT (10.0 12.0)" }
-  - match: { values.0.10: "LINESTRING (-97.154 25.996, -97.159 25.998, -97.181 25.991, -97.187 25.985)" }
+  - match: { values.0.9: "LINESTRING (-97.154 25.996, -97.159 25.998, -97.181 25.991, -97.187 25.985)" }
+  - match: { values.0.10: null }
   - match: { values.0.11: null }
   - match: { values.0.12: null }
   - match: { values.0.13: null }
-  - match: { values.0.14: null }
-  - match: { values.0.15: "foo bar baz" }
-  - match: { values.0.16: Alice }
-  - match: { values.0.17: "POINT (-97.15447 25.9961525)" }
+  - match: { values.0.14: "foo bar baz" }
+  - match: { values.0.15: Alice }
+  - match: { values.0.16: "POINT (-97.15447 25.9961525)" }
+  - match: { values.0.17: null }
   - match: { values.0.18: null }
   - match: { values.0.19: null }
   - match: { values.0.20: null }
   - match: { values.0.21: null }
   - match: { values.0.22: null }
-  - match: { values.0.23: null }
-  - match: { values.0.24: "LINESTRING (-377.03653 389.897676, -377.009051 389.889939)" }
-  - match: { values.0.25: 12 }
-  - match: { values.0.26: xy }
-  - match: { values.0.27: "foo bar" }
-  - match: { values.0.28: 3 }
+  - match: { values.0.23: "LINESTRING (-377.03653 389.897676, -377.009051 389.889939)" }
+  - match: { values.0.24: 12 }
+  - match: { values.0.25: xy }
+  - match: { values.0.26: "foo bar" }
+  - match: { values.0.27: 3 }
```

Comment:
Maybe add a yaml test that performs downsampling and queries data stream that has a downsampled backing index?